libs/dataviews-client/src/resources/get-resources.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  DatasetTypes,
+  DataviewDatasetConfig,
+  DataviewDatasetConfigParam,
+  EndpointId,
+  Resource,
+  ResourceStatus,
+} from '@globalfishingwatch/api-types'
+import { GeneratorType } from '@globalfishingwatch/layer-composer'
+import {
+  getDatasetConfigByDatasetType,
+  getDatasetConfigsByDatasetType,
+  UrlDataviewInstance,
+} from '../resolve-dataviews'
+import { resolveEndpoint } from '../resolve-endpoint'
+
+export const getResources = (
```

Comment:
Replaces `getDataviewsForResourceQuerying` and `resolveResourcesFromDatasetConfigs` in resolve-dataviews